src/ert/config/parsing/lark_parser.py

```diff
@@ -332,6 +332,7 @@ def _handle_includes(
             constraints = define_keyword()
             args = constraints.join_args(args)
             args = _substitute_args(args, constraints, defines)
+            args = constraints.apply_constraints(args, kw, os.getcwd())
```

Comment:
When testing this it seems like this throws an Error immediately, instead of appending it to a list of errors and then showing all Errors after parsing?  Would it not be preferred to keep the behaviour the same as in other similar cases?

Reply:
This is correct. The reason being that we need to resolve defines before we start parsing other content of the configuration file. With errors in the defines, we would likely spew misinformation regarding other valid content.